config/ufs/machines/config_machines.xml

```diff
@@ -451,7 +451,77 @@ This allows using a different mpirun command to launch unit tests
       <env name="W3NCO_LIBd">$ENV{NCEPLIBS_DIR}/lib/libw3nco_v2.0.6_d.a</env>
     </environment_variables>
   </machine>
-
+ <machine MACH="gaea">
+    <DESC>
+      port to gaea
+    </DESC>
+    <NODENAME_REGEX>gaea*</NODENAME_REGEX>
+    <OS>CNL</OS>
+    <COMPILERS>cray-intel</COMPILERS>
+    <MPILIBS>mpich</MPILIBS>
```

Comment:
I think that this should be mpt

Reply:
the module is called cray-mpich on gaea

Reply:
I'm not entirely sure on this one, we can try it.